### PR TITLE
Code Refactored for readability and maintainability

### PR DESCRIPTION
**Pull request by piyushjoshi01**
Improved Code Quality by removing code smells like Complex Conditional, Complex Method, Cyclically-dependent Modularization, Insufficient Modularization.

### CODE REVIEW

File: src/main/java/org/isf/serviceprinting/manager/PrintManager.java [hunk 1]
```diff
@@ -1,24 +1,3 @@
-/*
```
Comment (dbmalkovsky) [inline src/main/java/org/isf/serviceprinting/manager/PrintManager.java:1]: Come on....what are you doing removing licensing information?

**Comment by piyushjoshi01** [comment]
Sorry man, I will update it

File: src/main/java/org/isf/serviceprinting/manager/PrintManager.java (hunk 1 above)
Comment (piyushjoshi01) [reply src/main/java/org/isf/serviceprinting/manager/PrintManager.java]: Hi, Sorry for the inconvenience. I have added the the license information into PrintManager Class. Can you please review the changes I have made and if possible accept the pull request. The refactoring was part of my University assignment and I need to submit it today.

**Review by mwithi — CHANGES_REQUESTED** [review]
Hi @piyushjoshi01, thank you for your effort and contribution to this project. Upon reviewing it, we've identified a few concerns that need to be addressed before we can proceed with merging.

Firstly, there seem to be conflicts with existing files as indicated in the "PR automatic review" below. Additionally, we've noticed that there are some discrepancies between the test frameworks being used (Junit 4 versus Junit 5) which need to be rectified for consistency within our project standards.

Furthermore, while we appreciate the effort in refactoring the code into smaller methods, it appears that some of these methods lack comprehensive JavaDoc documentation. Additionally, we are concerned that the refactoring may introduce unnecessary overhead in terms of runtime due to increased method calls.

We truly value your contributions and believe that with these adjustments, your changes could significantly enhance our codebase. We encourage you to address these points and to have a look to the [Code Style](https://github.com/informatici/openhospital-core?tab=readme-ov-file#code-style) requsitions.

**Comment by piyushjoshi01** [comment]
I will look into this and get back to you.

**Comment by mwithi** [comment]
closing for no feedback for too long time

**Pull request closed without merging**